editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import PanelBody from 'components/panel/body';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+function PostStatus() {
+	return (
+		<PanelBody title={ __( 'Status & Visibility' ) }>
+			<div className="editor-sidebar-post-status__row">
```

Comment:
By className conventions we'd intentionally avoid reflecting nesting in the class name to allow the component to be more easily moved if necessary. I think this ought to be simplified to `editor-post-status__row`. Else maybe rename the parent directory to `sidebar-post-status`.